packages/frontend/src/widgets/header/ui/mobile-menu/MobileMenu.tsx

```diff
@@ -0,0 +1,90 @@
+import classNames from 'classnames'
+import { ButtonGeneral, LinkGeneral, SocialItem } from '@shared/ui'
+import { IconButtonClose, IconTelegramHeader, IconVkHeader, IconWhatsappHeader } from '@shared/ui/icons'
+import { dataNavLinks } from '../../model/dataHeader'
+import styles from './MobileMenu.module.scss'
+
+export const MobileMenu = ({ onClose }: { onClose: () => void }): JSX.Element => (
+  <div className={ styles['mobile-menu'] }
+    id='owner-menu-wrapper'
+  >
+    <div className={ styles['mobile-menu__inner'] }
+      id='owner-mobile-menu'
+    >
+      <ButtonGeneral
+        classProps={ classNames(styles['mobile-menu__btn-close']) }
+        font='s'
```

Comment:
Здесь другой шрифт у кнопки.
```suggestion
        font='m'
```